### PR TITLE
adds params

### PR DESCRIPTION
**Pull request by HarveyPeachey**
Resolves #NUMBER

**Overall change:**
_A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Review by amoore108 — APPROVED** [review]

File: src/app/lib/analyticsUtils/index.js [hunk 1]
```diff
@@ -619,3 +619,6 @@ export const getATIMarketingString = (href, campaignType) => {
 };
 
 export const LIBRARY_VERSION = 'simorgh';
+
+export const onOnionTld = () =>
+  onClient() ? window.location.host.endsWith('.onion') : false;
```
Comment (HarveyPeachey) [inline src/app/lib/analyticsUtils/index.js:624]: Ooh never knew endsWith was a method, really nice

**Review by HarveyPeachey — COMMENTED** [review]
Thanks for taking over on this, I can't approve but it LGTM

File: src/app/lib/analyticsUtils/index.js (hunk 1 above)
Comment (jroebu14) [reply src/app/lib/analyticsUtils/index.js:624]: there is startsWith too 🙂

Comment (jroebu14) [reply src/app/lib/analyticsUtils/index.js:624]: a bit nicer than a regex!

**Review by jroebu14 — APPROVED** [review]

**Pull request merged**